telco-examples/mgmt-cluster/aarch64/eib/mgmt-cluster-arm-singlenode.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: 1.0
+image:
+  imageType: iso
+  arch: aarch64
+  baseImage: SL-Micro.aarch64-6.0-Default-SelfInstall-GM2.install.iso
+  outputImageName: eib-mgmt-cluster-image-arm.iso
+operatingSystem:
+  isoConfiguration:
+    installDevice: /dev/sda
+  users:
+  - username: root
+    encryptedPassword: ${ROOT_PASSWORD}
+  packages:
+    packageList:
+    - jq
+    - openssh-server-config-rootlogin
+    sccRegistrationCode: ${SCC_REGISTRATION_CODE}
+kubernetes:
+  version: v1.30.3+rke2r1
+  helm:
+    charts:
+      - name: cert-manager
+        repositoryName: jetstack
+        version: 1.15.3
+        targetNamespace: cert-manager
+        valuesFile: certmanager.yaml
+        createNamespace: true
+        installationNamespace: kube-system
+      - name: longhorn-crd
+        version: 104.2.0+up1.7.1
+        repositoryName: rancher-charts
+        targetNamespace: longhorn-system
+        createNamespace: true
+        installationNamespace: kube-system
+      - name: longhorn
+        version: 104.2.0+up1.7.1
+        repositoryName: rancher-charts
+        targetNamespace: longhorn-system
+        createNamespace: true
+        installationNamespace: kube-system
+      - name: metal3
+        version: 0.9.1
+        repositoryName: suse-edge-upstream
+        targetNamespace: metal3-system
+        createNamespace: true
+        installationNamespace: kube-system
+        valuesFile: metal3.yaml
+      - name: neuvector-crd
+        version: 104.0.1+up2.7.9
+        repositoryName: rancher-charts
+        targetNamespace: neuvector
+        createNamespace: true
+        installationNamespace: kube-system
+        valuesFile: neuvector.yaml
+      - name: neuvector
+        version: 104.0.1+up2.7.9
+        repositoryName: rancher-charts
+        targetNamespace: neuvector
+        createNamespace: true
+        installationNamespace: kube-system
+        valuesFile: neuvector.yaml
+      - name: rancher
+        version: 2.9.1
+        repositoryName: rancher-prime
+        targetNamespace: cattle-system
+        createNamespace: true
+        installationNamespace: kube-system
+        valuesFile: rancher.yaml
+      - name: rancher-turtles-chart
```

Comment:
@hardys I need to use the github.io charts also for rancher-turtles because the build from oci helm chart 3.1 is failing due to the exec format